2024-08-06 20:37:50,304 INFO    StreamThr :2159681 [internal.py:wandb_internal():86] W&B internal server running at pid: 2159681, started at: 2024-08-06 20:37:50.303982
2024-08-06 20:37:50,305 DEBUG   HandlerThread:2159681 [handler.py:handle_request():146] handle_request: status
2024-08-06 20:37:50,305 INFO    WriterThread:2159681 [datastore.py:open_for_write():85] open: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/run-bk9cocci.wandb
2024-08-06 20:37:50,305 DEBUG   SenderThread:2159681 [sender.py:send():382] send: header
2024-08-06 20:37:50,305 DEBUG   SenderThread:2159681 [sender.py:send():382] send: run
2024-08-06 20:37:50,486 INFO    SenderThread:2159681 [dir_watcher.py:__init__():211] watching files in: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files
2024-08-06 20:37:50,486 INFO    SenderThread:2159681 [sender.py:_start_run_threads():1136] run started: bk9cocci with start time 1722947870.303987
2024-08-06 20:37:50,503 DEBUG   HandlerThread:2159681 [handler.py:handle_request():146] handle_request: check_version
2024-08-06 20:37:50,503 DEBUG   SenderThread:2159681 [sender.py:send_request():409] send_request: check_version
2024-08-06 20:37:50,613 DEBUG   HandlerThread:2159681 [handler.py:handle_request():146] handle_request: run_start
2024-08-06 20:37:50,625 DEBUG   HandlerThread:2159681 [system_info.py:__init__():32] System info init
2024-08-06 20:37:50,626 DEBUG   HandlerThread:2159681 [system_info.py:__init__():47] System info init done
2024-08-06 20:37:50,626 INFO    HandlerThread:2159681 [system_monitor.py:start():194] Starting system monitor
2024-08-06 20:37:50,626 INFO    SystemMonitor:2159681 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-06 20:37:50,626 INFO    HandlerThread:2159681 [system_monitor.py:probe():214] Collecting system info
2024-08-06 20:37:50,627 INFO    SystemMonitor:2159681 [interfaces.py:start():190] Started cpu monitoring
2024-08-06 20:37:50,628 INFO    SystemMonitor:2159681 [interfaces.py:start():190] Started disk monitoring
2024-08-06 20:37:50,629 INFO    SystemMonitor:2159681 [interfaces.py:start():190] Started gpu monitoring
2024-08-06 20:37:50,630 INFO    SystemMonitor:2159681 [interfaces.py:start():190] Started memory monitoring
2024-08-06 20:37:50,631 INFO    SystemMonitor:2159681 [interfaces.py:start():190] Started network monitoring
2024-08-06 20:37:50,640 DEBUG   HandlerThread:2159681 [system_info.py:probe():196] Probing system
2024-08-06 20:37:50,643 DEBUG   HandlerThread:2159681 [system_info.py:_probe_git():181] Probing git
2024-08-06 20:37:50,647 DEBUG   HandlerThread:2159681 [system_info.py:_probe_git():189] Probing git done
2024-08-06 20:37:50,647 DEBUG   HandlerThread:2159681 [system_info.py:probe():244] Probing system done
2024-08-06 20:37:50,647 DEBUG   HandlerThread:2159681 [system_monitor.py:probe():223] {'os': 'Linux-6.5.0-41-generic-x86_64-with-glibc2.35', 'python': '3.10.13', 'heartbeatAt': '2024-08-06T12:37:50.640713', 'startedAt': '2024-08-06T12:37:50.302408', 'docker': None, 'cuda': None, 'args': ('--local_rank=0', '--deepspeed', 'configs/ds_config_zero2_no_offload.json', '--model_name_or_path', '/home/rwkv/JL/model/qwen2-1.5b', '--data_path', '/home/rwkv/JL/model/MetaMathQA', '--dataset_field', 'query', 'response', '--dataset_split', 'train[:395000]', '--output_dir', '/home/rwkv/JL/out_model/Gemma-meta-2btest', '--num_train_epochs', '1', '--model_max_length', '512', '--per_device_train_batch_size', '2', '--gradient_accumulation_steps', '16', '--save_strategy', 'steps', '--save_steps', '1000', '--save_total_limit', '2', '--learning_rate', '2e-5', '--weight_decay', '0.', '--warmup_ratio', '0.03', '--logging_steps', '1', '--lr_scheduler_type', 'cosine', '--report_to', 'wandb', '--merge', 'True'), 'state': 'running', 'program': '/home/rwkv/JL/GMM/llama/gmm.py', 'codePathLocal': 'gmm.py', 'codePath': 'llama/gmm.py', 'git': {'remote': 'https://github.com/JL-er/GMM.git', 'commit': None}, 'email': '997529190@qq.com', 'root': '/home/rwkv/JL/GMM', 'host': 'rwkv-Z790M-AORUS-ELITE-AX', 'username': 'rwkv', 'executable': '/home/rwkv/miniconda3/bin/python', 'cpu_count': 20, 'cpu_count_logical': 28, 'cpu_freq': {'current': 1346.319857142857, 'min': 800.0, 'max': 5000.0}, 'cpu_freq_per_core': [{'current': 2158.764, 'min': 800.0, 'max': 5500.0}, {'current': 2113.22, 'min': 800.0, 'max': 5500.0}, {'current': 2198.964, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 2199.079, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 2187.174, 'min': 800.0, 'max': 5500.0}, {'current': 2194.602, 'min': 800.0, 'max': 5600.0}, {'current': 2186.326, 'min': 800.0, 'max': 5600.0}, {'current': 2199.276, 'min': 800.0, 'max': 5600.0}, {'current': 800.0, 'min': 800.0, 'max': 5600.0}, {'current': 1910.065, 'min': 800.0, 'max': 5500.0}, {'current': 2185.779, 'min': 800.0, 'max': 5500.0}, {'current': 2182.215, 'min': 800.0, 'max': 5500.0}, {'current': 800.0, 'min': 800.0, 'max': 5500.0}, {'current': 1440.13, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 1961.353, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}, {'current': 800.0, 'min': 800.0, 'max': 4300.0}], 'disk': {'/': {'total': 916.2753829956055, 'used': 443.3819694519043}}, 'gpu': 'NVIDIA GeForce RTX 4090', 'gpu_count': 1, 'gpu_devices': [{'name': 'NVIDIA GeForce RTX 4090', 'memory_total': 25757220864}], 'memory': {'total': 62.5532341003418}}
2024-08-06 20:37:50,647 INFO    HandlerThread:2159681 [system_monitor.py:probe():224] Finished collecting system info
2024-08-06 20:37:50,647 INFO    HandlerThread:2159681 [system_monitor.py:probe():227] Publishing system info
2024-08-06 20:37:50,647 DEBUG   HandlerThread:2159681 [system_info.py:_save_pip():52] Saving list of pip packages installed into the current environment
2024-08-06 20:37:50,648 DEBUG   HandlerThread:2159681 [system_info.py:_save_pip():68] Saving pip packages done
2024-08-06 20:37:50,648 DEBUG   HandlerThread:2159681 [system_info.py:_save_conda():75] Saving list of conda packages installed into the current environment
2024-08-06 20:37:51,486 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/conda-environment.yaml
2024-08-06 20:37:51,486 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/requirements.txt
2024-08-06 20:37:52,332 DEBUG   HandlerThread:2159681 [system_info.py:_save_conda():87] Saving conda packages done
2024-08-06 20:37:52,332 DEBUG   HandlerThread:2159681 [system_info.py:_save_code():90] Saving code
2024-08-06 20:37:52,333 DEBUG   HandlerThread:2159681 [system_info.py:_save_code():111] Saving code done
2024-08-06 20:37:52,333 DEBUG   HandlerThread:2159681 [system_info.py:_save_patches():128] Saving git patches
2024-08-06 20:37:52,336 ERROR   HandlerThread:2159681 [system_info.py:_save_patches():169] Error generating diff: Reference at 'refs/remotes/origin/main' does not exist
2024-08-06 20:37:52,336 DEBUG   HandlerThread:2159681 [system_info.py:_save_patches():170] Saving git patches done
2024-08-06 20:37:52,336 INFO    HandlerThread:2159681 [system_monitor.py:probe():229] Finished publishing system info
2024-08-06 20:37:52,337 DEBUG   SenderThread:2159681 [sender.py:send():382] send: files
2024-08-06 20:37:52,337 INFO    SenderThread:2159681 [sender.py:_save_file():1392] saving file wandb-metadata.json with policy now
2024-08-06 20:37:52,337 INFO    SenderThread:2159681 [sender.py:_save_file():1392] saving file code/llama/gmm.py with policy now
2024-08-06 20:37:52,340 DEBUG   HandlerThread:2159681 [handler.py:handle_request():146] handle_request: internal_messages
2024-08-06 20:37:52,340 DEBUG   HandlerThread:2159681 [handler.py:handle_request():146] handle_request: stop_status
2024-08-06 20:37:52,340 DEBUG   SenderThread:2159681 [sender.py:send():382] send: telemetry
2024-08-06 20:37:52,340 DEBUG   SenderThread:2159681 [sender.py:send_request():409] send_request: stop_status
2024-08-06 20:37:52,486 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/conda-environment.yaml
2024-08-06 20:37:52,486 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/wandb-metadata.json
2024-08-06 20:37:52,486 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/code/llama/gmm.py
2024-08-06 20:37:52,486 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/code/llama
2024-08-06 20:37:52,486 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/code
2024-08-06 20:37:52,532 DEBUG   SenderThread:2159681 [sender.py:send():382] send: config
2024-08-06 20:37:52,532 DEBUG   SenderThread:2159681 [sender.py:send():382] send: metric
2024-08-06 20:37:52,532 DEBUG   SenderThread:2159681 [sender.py:send():382] send: telemetry
2024-08-06 20:37:52,532 DEBUG   SenderThread:2159681 [sender.py:send():382] send: metric
2024-08-06 20:37:52,532 WARNING SenderThread:2159681 [sender.py:send_metric():1343] Seen metric with glob (shouldn't happen)
2024-08-06 20:37:52,532 DEBUG   SenderThread:2159681 [sender.py:send():382] send: telemetry
2024-08-06 20:37:52,532 DEBUG   SenderThread:2159681 [sender.py:send():382] send: telemetry
2024-08-06 20:37:52,532 DEBUG   SenderThread:2159681 [sender.py:send():382] send: config
2024-08-06 20:37:52,600 ERROR   wandb-upload_0:2159681 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/bk9cocci/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T123752Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=40888e35be0018414218dafbcf3d9d785834a56d518a4090f8f8ad34640ab703: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/bk9cocci/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T123752Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=40888e35be0018414218dafbcf3d9d785834a56d518a4090f8f8ad34640ab703
2024-08-06 20:37:52,600 ERROR   wandb-upload_0:2159681 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5189', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 20:37:52,600 ERROR   wandb-upload_0:2159681 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/bk9cocci/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/bk9cocci/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E923F626D4FFE0</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 20:37:52,703 ERROR   wandb-upload_1:2159681 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/bk9cocci/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T123752Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=dd5902c6612036fd280ef59266d691b9d574dbb79b99b41db6e20166486a3c97: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/bk9cocci/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T123752Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=dd5902c6612036fd280ef59266d691b9d574dbb79b99b41db6e20166486a3c97
2024-08-06 20:37:52,703 ERROR   wandb-upload_1:2159681 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 20:37:52,703 ERROR   wandb-upload_1:2159681 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/bk9cocci/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/bk9cocci/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E923F62933F5C2</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 20:37:53,486 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/output.log
2024-08-06 20:37:53,679 ERROR   wandb-upload_0:2159681 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/bk9cocci/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T123752Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=40888e35be0018414218dafbcf3d9d785834a56d518a4090f8f8ad34640ab703: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/bk9cocci/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T123752Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=40888e35be0018414218dafbcf3d9d785834a56d518a4090f8f8ad34640ab703
2024-08-06 20:37:53,679 ERROR   wandb-upload_0:2159681 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5189', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 20:37:53,679 ERROR   wandb-upload_0:2159681 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/bk9cocci/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/bk9cocci/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E923F6671A7E52</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 20:37:53,922 ERROR   wandb-upload_1:2159681 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/bk9cocci/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T123752Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=dd5902c6612036fd280ef59266d691b9d574dbb79b99b41db6e20166486a3c97: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/bk9cocci/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T123752Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=dd5902c6612036fd280ef59266d691b9d574dbb79b99b41db6e20166486a3c97
2024-08-06 20:37:53,923 ERROR   wandb-upload_1:2159681 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 20:37:53,923 ERROR   wandb-upload_1:2159681 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/bk9cocci/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/bk9cocci/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E923F674B387A7</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 20:37:53,923 INFO    wandb-upload_1:2159681 [retry.py:__call__():172] Retry attempt failed:
Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/bk9cocci/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T123752Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=dd5902c6612036fd280ef59266d691b9d574dbb79b99b41db6e20166486a3c97

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/lib/retry.py", line 131, in __call__
    result = self._call_fn(*args, **kwargs)
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2785, in upload_file
    raise _e.with_traceback(sys.exc_info()[2])
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/wandb/sdk/internal/internal_api.py", line 2759, in upload_file
    response.raise_for_status()
  File "/home/rwkv/miniconda3/lib/python3.10/site-packages/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
wandb.sdk.lib.retry.TransientError: None
2024-08-06 20:37:54,466 DEBUG   HandlerThread:2159681 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 20:37:54,467 DEBUG   SenderThread:2159681 [sender.py:send():382] send: metric
2024-08-06 20:37:54,467 DEBUG   SenderThread:2159681 [sender.py:send():382] send: metric
2024-08-06 20:37:54,467 DEBUG   SenderThread:2159681 [sender.py:send():382] send: metric
2024-08-06 20:37:54,467 DEBUG   SenderThread:2159681 [sender.py:send():382] send: metric
2024-08-06 20:37:54,467 DEBUG   SenderThread:2159681 [sender.py:send():382] send: history
2024-08-06 20:37:54,468 DEBUG   SenderThread:2159681 [sender.py:send_request():409] send_request: summary_record
2024-08-06 20:37:54,468 INFO    SenderThread:2159681 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 20:37:54,486 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_created():271] file/dir created: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/wandb-summary.json
2024-08-06 20:37:55,469 DEBUG   HandlerThread:2159681 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 20:37:55,486 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/output.log
2024-08-06 20:37:56,127 ERROR   wandb-upload_0:2159681 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/bk9cocci/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T123752Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=40888e35be0018414218dafbcf3d9d785834a56d518a4090f8f8ad34640ab703: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/bk9cocci/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T123752Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=40888e35be0018414218dafbcf3d9d785834a56d518a4090f8f8ad34640ab703
2024-08-06 20:37:56,127 ERROR   wandb-upload_0:2159681 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5189', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 20:37:56,127 ERROR   wandb-upload_0:2159681 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/bk9cocci/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/bk9cocci/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E923F6F9001E91</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 20:37:56,216 DEBUG   HandlerThread:2159681 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 20:37:56,217 DEBUG   SenderThread:2159681 [sender.py:send():382] send: history
2024-08-06 20:37:56,217 DEBUG   SenderThread:2159681 [sender.py:send_request():409] send_request: summary_record
2024-08-06 20:37:56,218 INFO    SenderThread:2159681 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 20:37:56,222 ERROR   wandb-upload_1:2159681 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/bk9cocci/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T123752Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=dd5902c6612036fd280ef59266d691b9d574dbb79b99b41db6e20166486a3c97: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/bk9cocci/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T123752Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=dd5902c6612036fd280ef59266d691b9d574dbb79b99b41db6e20166486a3c97
2024-08-06 20:37:56,222 ERROR   wandb-upload_1:2159681 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 20:37:56,222 ERROR   wandb-upload_1:2159681 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/bk9cocci/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/bk9cocci/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E923F6F9CBB5F3</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 20:37:56,487 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/wandb-summary.json
2024-08-06 20:37:56,487 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/output.log
2024-08-06 20:37:57,487 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/output.log
2024-08-06 20:37:57,841 DEBUG   HandlerThread:2159681 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 20:37:57,842 DEBUG   SenderThread:2159681 [sender.py:send():382] send: history
2024-08-06 20:37:57,842 DEBUG   SenderThread:2159681 [sender.py:send_request():409] send_request: summary_record
2024-08-06 20:37:57,842 INFO    SenderThread:2159681 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 20:37:58,487 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/wandb-summary.json
2024-08-06 20:37:58,487 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/output.log
2024-08-06 20:37:59,487 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/output.log
2024-08-06 20:37:59,611 DEBUG   HandlerThread:2159681 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 20:37:59,611 DEBUG   SenderThread:2159681 [sender.py:send():382] send: history
2024-08-06 20:37:59,611 DEBUG   SenderThread:2159681 [sender.py:send_request():409] send_request: summary_record
2024-08-06 20:37:59,612 INFO    SenderThread:2159681 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 20:38:00,303 ERROR   wandb-upload_0:2159681 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/bk9cocci/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T123752Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=40888e35be0018414218dafbcf3d9d785834a56d518a4090f8f8ad34640ab703: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/bk9cocci/wandb-metadata.json?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T123752Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=40888e35be0018414218dafbcf3d9d785834a56d518a4090f8f8ad34640ab703
2024-08-06 20:38:00,303 ERROR   wandb-upload_0:2159681 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '5189', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 20:38:00,303 ERROR   wandb-upload_0:2159681 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/bk9cocci/wandb-metadata.json</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/bk9cocci/wandb-metadata.json</Resource><Region>wandb-local</Region><RequestId>17E923F7F1AF4BCA</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 20:38:00,487 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/wandb-summary.json
2024-08-06 20:38:00,488 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/output.log
2024-08-06 20:38:00,612 DEBUG   HandlerThread:2159681 [handler.py:handle_request():146] handle_request: status_report
2024-08-06 20:38:00,807 ERROR   wandb-upload_1:2159681 [internal_api.py:upload_file():2761] upload_file exception http://dian.sea-group.org:9098/local-files/mobius/huggingface/bk9cocci/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T123752Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=dd5902c6612036fd280ef59266d691b9d574dbb79b99b41db6e20166486a3c97: 503 Server Error: Service Unavailable for url: http://dian.sea-group.org:9098/local-files/mobius/huggingface/bk9cocci/code/llama/gmm.py?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=FS%2BimP8205gHBuibMeAr0A%3D%3D%2F20240806%2Fwandb-local%2Fs3%2Faws4_request&X-Amz-Date=20240806T123752Z&X-Amz-Expires=86400&X-Amz-SignedHeaders=host&X-User=mobius&X-Amz-Signature=dd5902c6612036fd280ef59266d691b9d574dbb79b99b41db6e20166486a3c97
2024-08-06 20:38:00,807 ERROR   wandb-upload_1:2159681 [internal_api.py:upload_file():2763] upload_file request headers: {'User-Agent': 'python-requests/2.31.0', 'Accept-Encoding': 'gzip, deflate, br, zstd', 'Accept': '*/*', 'Connection': 'keep-alive', 'Content-Length': '14100', 'Authorization': 'Basic dXNlcjpsb2NhbC0yNWIwMmYyNWUzNGUzOTlkYmI1OWJjNjFlZWVmY2RkZTYzNDRhOGI2'}
2024-08-06 20:38:00,807 ERROR   wandb-upload_1:2159681 [internal_api.py:upload_file():2765] upload_file response body: b'<?xml version="1.0" encoding="UTF-8"?>\n<Error><Code>SlowDown</Code><Message>Resource requested is unreadable, please reduce your request rate</Message><Key>mobius/huggingface/bk9cocci/code/llama/gmm.py</Key><BucketName>local-files</BucketName><Resource>/local-files/mobius/huggingface/bk9cocci/code/llama/gmm.py</Resource><Region>wandb-local</Region><RequestId>17E923F8060DAC7F</RequestId><HostId>18bf63b7-01d2-4a3f-8eda-6b508ac9c4f3</HostId></Error>'
2024-08-06 20:38:01,338 DEBUG   HandlerThread:2159681 [handler.py:handle_request():146] handle_request: partial_history
2024-08-06 20:38:01,338 DEBUG   SenderThread:2159681 [sender.py:send():382] send: history
2024-08-06 20:38:01,338 DEBUG   SenderThread:2159681 [sender.py:send_request():409] send_request: summary_record
2024-08-06 20:38:01,339 INFO    SenderThread:2159681 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 20:38:01,488 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/wandb-summary.json
2024-08-06 20:38:01,488 INFO    Thread-12 :2159681 [dir_watcher.py:_on_file_modified():288] file/dir modified: /home/rwkv/JL/GMM/llama/wandb/run-20240806_203750-bk9cocci/files/output.log
2024-08-06 20:38:01,850 DEBUG   SenderThread:2159681 [sender.py:send():382] send: exit
2024-08-06 20:38:01,850 INFO    SenderThread:2159681 [sender.py:send_exit():589] handling exit code: 255
2024-08-06 20:38:01,850 INFO    SenderThread:2159681 [sender.py:send_exit():591] handling runtime: 11
2024-08-06 20:38:01,850 INFO    SenderThread:2159681 [sender.py:_save_file():1392] saving file wandb-summary.json with policy end
2024-08-06 20:38:01,851 INFO    SenderThread:2159681 [sender.py:send_exit():597] send defer
2024-08-06 20:38:01,851 DEBUG   HandlerThread:2159681 [handler.py:handle_request():146] handle_request: defer
2024-08-06 20:38:01,851 INFO    HandlerThread:2159681 [handler.py:handle_request_defer():172] handle defer: 0
2024-08-06 20:38:01,851 DEBUG   SenderThread:2159681 [sender.py:send_request():409] send_request: defer
2024-08-06 20:38:01,851 INFO    SenderThread:2159681 [sender.py:send_request_defer():613] handle sender defer: 0
2024-08-06 20:38:01,851 INFO    SenderThread:2159681 [sender.py:transition_state():617] send defer: 1
2024-08-06 20:38:01,851 DEBUG   HandlerThread:2159681 [handler.py:handle_request():146] handle_request: defer
2024-08-06 20:38:01,851 INFO    HandlerThread:2159681 [handler.py:handle_request_defer():172] handle defer: 1
2024-08-06 20:38:01,851 DEBUG   SenderThread:2159681 [sender.py:send_request():409] send_request: defer
2024-08-06 20:38:01,851 INFO    SenderThread:2159681 [sender.py:send_request_defer():613] handle sender defer: 1
2024-08-06 20:38:01,851 INFO    SenderThread:2159681 [sender.py:transition_state():617] send defer: 2
2024-08-06 20:38:01,851 DEBUG   HandlerThread:2159681 [handler.py:handle_request():146] handle_request: defer
2024-08-06 20:38:01,851 INFO    HandlerThread:2159681 [handler.py:handle_request_defer():172] handle defer: 2
2024-08-06 20:38:01,851 INFO    HandlerThread:2159681 [system_monitor.py:finish():203] Stopping system monitor
2024-08-06 20:38:01,851 DEBUG   SystemMonitor:2159681 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-06 20:38:01,851 INFO    HandlerThread:2159681 [interfaces.py:finish():202] Joined cpu monitor
2024-08-06 20:38:01,851 DEBUG   SystemMonitor:2159681 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-08-06 20:38:01,851 INFO    HandlerThread:2159681 [interfaces.py:finish():202] Joined disk monitor
2024-08-06 20:38:01,852 DEBUG   SystemMonitor:2159681 [system_monitor.py:_start():183] Publishing last batch of metrics
